projects/storefrontlib/src/cms-components/myaccount/order/cancellations-returns/amend-order-actions/amend-order-actions.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="col-xs-12 col-md-4 col-lg-3">
+  <a
+    [routerLink]="
+      {
+        cxRoute: backRoute,
+        params: { code: orderCode }
+      } | cxUrl
+    "
+    class="btn btn-block btn-action"
+  >
+    {{ 'common.back' | cxTranslate }}
+  </a>
+</div>
+<div class="col-xs-12 col-md-4 col-lg-3">
+  <a
+    *ngIf="forwardRoute"
+    class="btn btn-block btn-primary"
+    [class.disabled]="!isValid"
+    [routerLink]="
+      isValid
+        ? ({
```

Comment:
Do you still need this router link part even with the disabled?

Reply:
Good one!